work_with_builds/using-builds.adoc

```diff
@@ -0,0 +1,27 @@
+:_content-type: ASSEMBLY
+[id="using-builds"]
+include::_attributes/common-attributes.adoc[]
+= Using {builds-shortname}
+:context: using-builds
+
+toc::[]
+
+
+After installing {builds-shortname}, you can create a `buildah` or `source-to-image` build for use. You can also delete resources that are not required for a build. 
```

Comment:
If we can explicitly specify what resources we are talking to be deleted here, that'll be great.